app/checkout/[userId]/page.tsx

```diff
@@ -56,6 +57,24 @@ export default function CheckOut({ params }: CheckOutProps) {
         router.push(`/`);
     };
 
+    const onChange = (visitData: Visit) => {
+        setVisitFormData(visitData);
+    };
+
+    const saveVisit = async (visitData: Visit) => {
+        await updateVisit(visitData, params.userId);
+        const client = await updateClient({
+            ...clientData,
+        });
+        updateVisitCache(params.userId, visitData);
+        updateClientCache(client);
+        setAlert({
+            message: `Successfully updated visit for ${clientData?.firstName}`,
+            type: 'success',
+        });
+        router.push(`/checkedin`);
```

Comment:
Is there a reason to route to checkedin page? Could we stay in the checkout page and close the modal after saving instead. User might want to do more editing and saving?

Reply:
My initial thought was that the user might want to work on other checkedin clients after saving. Sure we can stay in the checkout page for the same client in case there's a need to continue editing.